### ISSUE TITLE
[1주차] DEC: Unsupervised Deep Embedding for Clustering Analysis

### ISSUE THREAD

**Issue by jwkanggist** [jwkanggist/self-supervised-learning-narratives-1#3]
# Keywords
- DEC, clustering

# TL;DR
- 단순히 feature 간의 거리를 이용하여 clustering을 하는 것이 아니라, 데이터의 feature representations와 cluster assignments를 Neural Network를 통해 동시에 학습할 수 있는가?
    - YES
        - target likelihood distribution을 student-t로 모델링하고
        - backprob을 통해서 student-t의 mean과  encoder 파라미터를 동시에 업데이트

# Abstract
Clustering is central to many data-driven application domains and has been studied extensively in terms of distance functions and grouping algorithms. Relatively little work has focused on learning representations for clustering. In this paper, we propose Deep Embedded Clustering (DEC), a method that simultaneously learns feature representations and cluster assignments using deep neural networks. DEC learns a mapping from the data space to a lower-dimensional feature space in which it iteratively optimizes a clustering objective. Our experimental evaluations on image and text corpora show significant improvement over state-of-the-art methods.

# Paper link
https://proceedings.mlr.press/v48/xieb16.pdf

# Presentation link
https://www.notion.so/jwkangmarco/Unsupervised-Deep-Embedding-for-Clustering-Analysis-398f1d4910b5446981fd054395798fcd

# video link
